package.json

```diff
@@ -158,7 +158,7 @@
 		"bbb-promise": "^1.2.0",
 		"bcryptjs": "*",
 		"body-parser": "^1.15.2",
-		"bson": "^4.6.0",
+		"bson": "^5.3.0",
```

Comment:
TODO (out of scope): uninstall bson as stand-alone library
in nest we should use it via mikro-orm, in feathers via mongoose.